content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/index.mdx

```diff
@@ -0,0 +1,120 @@
+- Access to a MongoDB 4.2+ Server. We recommend using [MongoDB Atlas](https://www.mongodb.com/cloud/atlas)
+
+
+> See [System requirements](../../../../reference/system-requirements) for exact version requirements.
+
+Make sure you have your database [connection URL](../../../../reference/database-reference/connection-urls) at hand. If you don't have a database server running and just want to explore Prisma, check out the [Quickstart](../../../quickstart).
+
+<!--
+Alternatively you can [set up a free PostgreSQL database on Heroku](https://dev.to/prisma/how-to-setup-a-free-postgresql-database-on-heroku-1dc1) and use it in this guide.
+-->
+
+## Create project setup
+
+As a first step, create a project directory and navigate into it:
+
+```terminal copy
+mkdir hello-prisma
+cd hello-prisma
+```
+
+<SwitchTech technologies={['typescript', '*']}>
+
+Next, initialize a TypeScript project and add the Prisma CLI as a development dependency to it:
+
+```terminal copy
+npm init -y
+npm install prisma typescript ts-node @types/node --save-dev
+```
+
+This creates a `package.json` with an initial setup for your TypeScript app.
+
+Next, create a `tsconfig.json` file and add the following configuration to it:
+
+```json file=tsconfig.json copy
+{
+  "compilerOptions": {
+    "sourceMap": true,
+    "outDir": "dist",
+    "strict": true,
+    "lib": ["esnext"],
+    "esModuleInterop": true
+  }
+}
+```
+
+</SwitchTech>
+
+<SwitchTech technologies={['node', '*']}>
+
+Next, initialize a Node.js project and add the Prisma CLI as a development dependency to it:
+
+```terminal copy
+npm init -y
+npm install prisma --save-dev
+```
+
+This creates a `package.json` with an initial setup for a Node.js app.
+
+</SwitchTech>
+
+You can now invoke the Prisma CLI by prefixing it with `npx`:
+
+```terminal
+npx prisma
+```
+
+Next, set up your Prisma project by creating your [Prisma schema](../../../../concepts/components/prisma-schema) file with the following command:
+
+```terminal copy
+npx prisma init
+```
+
+This command created a new directory called `prisma` which contains a file named `schema.prisma` and a `.env` file in the root of the project. `schema.prisma` contains the Prisma schema with your database connection and the Prisma Client generator.
+`.env` is a [dotenv](https://github.com/motdotla/dotenv) file for defining environment variables (used for your database connection).
+
+
+<SwitchTech technologies={['node', 'mongodb']}>
+
+<NavigationLinksContainer isFirstLink>
+
+<ButtonLink color="dark" type="primary" href="./mongodb/connect-your-database-node-mongodb" arrow>
+    Connect your database
+</ButtonLink>
+
+</NavigationLinksContainer>
+
+</SwitchTech>
+
+<SwitchTech technologies={['typescript', 'mongodb']}>
+
+<NavigationLinksContainer isFirstLink>
+
+<ButtonLink color="dark" type="primary" href="./mongodb/connect-your-database-typescript-mongodb" arrow>
+    Connect your database
+</ButtonLink>
+
+</NavigationLinksContainer>
+
+</SwitchTech>
```

Comment:
I think these buttons are not visible right now - the tutorial feels much to short.

Reply:
They are visible, see the screenshot below. They are in fact just 1 button, depending on if JS or TS is chosen

![mongo-button](https://user-images.githubusercontent.com/22930449/124758449-e34e9600-df2e-11eb-9ccb-54757a005f0a.PNG)

Reply:
Hmm then I maybe missed that somehow - I remember looking this up after feeling lost on a page. The buttons look great and make a lot of sense: https://deploy-preview-1900--prisma2-docs.netlify.app/docs/getting-started/setup-prisma/start-from-scratch/mongodb/connect-your-database-typescript-mongodb/